phpstan.neon

```diff
@@ -10,4 +11,23 @@ parameters:
         - tests
     excludePaths:
         - tests/Fixtures
+    treatPhpDocTypesAsCertain: false
```

Comment:
`true` is default value, so it is in fact change from `true` to `false`.

Why? Because of the PHPDoc in [`Token:__construct`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.1.0/src/Tokenizer/Token.php#L56) - PHPdoc claims it's `array` or `string` while in code we throw exception when none of these - we can refactor it later (I guess after some discussion), but let's focus of introducing strict rules in this PR.

Reply:
Not sure I understand. In this case, the PHPDoc is correct, isn't it? It could be precise though.

Reply:
Either we trust that PHPDoc or not:
- if we do, we should remove the [`else`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.1.0/src/Tokenizer/Token.php#L83-L89)
- if we don't - we can remove it, or set `treatPhpDocTypesAsCertain` to `false` as I did as this is not changing the code we already have

I'd definitely want to go with the 1st solution, but it can potentially broke (which actually be a good thing) some third party library for custom tokens (will not break anything in this repo).

Reply:
I would remove the `else` as well. Nowadays, we can assume static analysis tools are widespread enough to avoid such redundant checks.

Reply:
@julienfalque done